src/js/structs/JobActiveRun.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment';
+
+import Item from './Item';
+import JobTaskList from './JobTaskList';
+
+module.exports = class JobActiveRun extends Item {
+  getDateCreated() {
+    let dateCreated = this.get('createdAt');
+
+    if (dateCreated != null) {
+      return moment(dateCreated.toUpperCase());
```

Comment:
can we put this inside of a Util where it then returns the `moment().valueOf()` since that's what you'll be using? the reason for this is because right now our use of moment is limited to the DateUtil and if we ever decide that moment is terrible and needs to be replaced, we can replace it in one place